src/Model/Solid/Linear/Isotropic.H

```diff
@@ -95,8 +95,26 @@ public:
         out << ddw;
     }
 
+    Set::Scalar W2(const Set::Matrix & stress)
+    {
+        return ( 0.5 * stress.transpose() * (ddw_inv * stress) ).trace();
+    }
+    Set::Matrix DW2(const Set::Matrix & stress) 
+    {
+        return ddw_inv * stress;
+    }
+    Set::Scalar Wvol(const Set::Scalar eps_tr)
+    {
+        return 0.5*ddw.Lambda()*eps_tr*eps_tr;
+    }
+    Set::Scalar Wdev(const Set::Matrix eps)
+    {
+        return ddw.Mu()*std::max((eps*eps).trace(),0.0);
+    }
+
```

Comment:
@vinagr I'm not a fan of putting in these extra functions. `Linear::Isotropic` gets used a lot of places and I want to keep the formulation as simple as possible. Is it really necessary to have them? Can you just calculate these outside where they are used in `Fracture`?